iModelCore/ECPresentation/Source/Shared/ValueHelpers.cpp

```diff
@@ -590,6 +590,24 @@ ECValue ValueHelpers::GetECValueFromJson(PrimitiveType type, RapidJsonValueCR js
     return value;
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+bvector<ECValue> ValueHelpers::GetECValueSetFromJson(PrimitiveType type, RapidJsonValueCR json)
+    {
+    bvector<ECValue> ecValues;
+    for (rapidjson::SizeType i = 0; i < json.Size(); i++)
+        {
+        ECValue value = GetECValueFromJson(type, json[i]);
+        if (value.IsNull())
+            ecValues.push_back(ECValue(type));

```

Comment:
this is interesting. is `ECValue(PrimitiveType)` any different from just `ECValue()`?

Reply:
There are cases where the value can be null and still have a type. I think there was at least one test case where such was the only value in the set, so I somehow needed to get the type.
That's why in other places I check for `IsPrimitive()`, not `IsNull()`.
Can double check if you think this is not how it's supposed to be.

Reply:
Honestly, I don't like the idea of having different types of NULL. I think there's a problem if null without type doesn't equal null with a type.

Reply:
Fixed the issue.